LICENSE

```diff
@@ -173,3 +172,30 @@
       defend, and hold each Contributor harmless for any liability
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
The year 2018 needs to be filled in?
Also, what should be the copyright owner?  Please see this AUTHORS file:
https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/AUTHORS

Reply:
No, actually it shouldn't be filled in. According to the OSPO office the license file should be text version of https://www.apache.org/licenses/LICENSE-2.0, unchanged.

Reply:
It should be filled in in the source code headers, though.

Reply:
I see, we have those.  Thanks for responding!